portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -95,7 +109,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
         commentsList.clear();
     }
     else if (maxComments > 0 && maxComments < commentsList.size()){
-        commentsList.subList(0, commentsList.size() - maxComments).clear();
+        commentsList.subList(maxComments, commentsList.size()).clear();
```

Comment:
Do you want to keep the newest N comments or the oldest N comments (where N = maxComments)? 
It looks like what you have now will keep the newest N comments, and what you had before will keep the oldest N comments, since `comments` is sorted in reverse order of timestamp (bigger timestamps first and smaller timestamps last). 

Can you add a comment to the code as well as update the description of the PR so it's clear what your choice is regarding whether to keep the "newest or oldest N comments"?

Reply:
Hi Vivek, I change it so only the newest comments will be kept.